web/packages/design/src/theme/utils/colorManipulator.js

```diff
@@ -133,9 +133,9 @@ export function decomposeColor(color) {
 /**
  * Converts a color object with type and values to a string.
  *
- * @param {object} color - Decomposed color
- * @param {string} color.type - One of: 'rgb', 'rgba', 'hsl', 'hsla'
- * @param {array} color.values - [n,n,n] or [n,n,n,n]
+ * @param {object} bgColor - Decomposed color
+ * @param {string} bgColor.type - One of: 'rgb', 'rgba', 'hsl', 'hsla'
+ * @param {array} bgColor.values - [n,n,n] or [n,n,n,n]
```

Comment:
Is this a find and replace all incident or is it indeed supposed to be called `bgColor`? ;)

Reply:
What the...